services/QuillLMS/app/workers/save_activity_session_old_concept_results_worker.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class SaveActivitySessionOldConceptResultsWorker
+  include Sidekiq::Worker
+
+  def perform(concept_results_hashes)
+    ocr_ids = []
+    OldConceptResult.transaction do
+      concept_results_hashes.each do |concept_result_hash|
+        ocr_ids.append(OldConceptResult.create!(concept_result_hash).id)
```

Comment:
I think it would be useful to have a spec that has create! raise an error so that your expectations on     `SaveActivitySessionConceptResultsWorker.perform_async(ocr_ids)` being called are transparent.

Reply:
Which also kind of raises the question, should failed create! be sent to ErrorNotifier like in    SaveActivitySessionConceptResultsWorker?

Reply:
I've added a spec, so let me know what you think.  As to your follow-up question, no, actually.  Errors are automatically sent to Sentry and NewRelic when they're raised.  In the other worker, we're not actually raising the error (because we wanted to allow some weird retry logic that I suspect won't be necessary.  That is: I don't think that code path ever triggers.  But, short version: `ErrorNotifier` isn't necessary here because we're actually raising the error.

Reply:
Got it.  Thanks for the explanation.